GenderPayGap.WebUI/wwwroot/styles/components/_admin-search-box.scss

```diff
@@ -4,25 +4,28 @@
 
     .admin-search-button {
         background-color: $govuk-input-border-colour;
-        border: 0;
         border-radius: 0;
         cursor: pointer;
         position: relative;
         padding: 0;
         width: 40px;
         height: 40px;
-        background-image: url('/assets/images/search-button-white-and-black.png');
-        background-repeat: no-repeat;
-        background-position: 2px 50%;
-        text-indent: -5000px;
+        border-style: solid;
+        border-width: 4px 4px 4px 2px;
+        border-color: govuk-colour("black");
         overflow: hidden;
 
+        .svg {

```

Comment:
(this is ok for now, but...)
Ideally, we'd either:
* Use a descriptive class name
So, we'd be saying "Make any search icons within the admin-search-button have this style"
e.g.
```
.search-icon {
}
```
* Or, just use the element name
So, we'd be saying "Make any svg elements within the admin-search-button have this style"
e.g.
```
svg {
}

Having an svg who's class is "svg" doesn't add much meaning
```

Reply:
That's a fair point